### PR TITLE
tests for vimeo controller

### PR DESCRIPTION
**Pull request by ktrops**

### CODE REVIEW

File: spec/factories/feeds.rb [hunk 1]
```diff
@@ -5,4 +5,20 @@
...
+  factory :user_vimeo, class: Feed do
+    id 741273
+    name "Riley Blakeway"
+    avatar nil
+    platform "vimeo"
+    platform_feed_id 741273
+  end
+
+  factory :invalid_vimeo_feed, class: Feed do
+    name "I like videos"
```
Comment (Brennaleker) [inline spec/factories/feeds.rb:18]: I love this name!

**Comment by Brennaleker** [comment]
looks great! Thank you for writing tests!

File: spec/controllers/sessions_controller_spec.rb [hunk 2]
```diff
@@ -39,4 +39,13 @@
     end
   end
 
+  describe "#destory" do 
+    let(:user) { FactoryGirl.create(:au_user)}
```
Comment (vikshab) [inline spec/controllers/sessions_controller_spec.rb:43]: I think you can just do like this: `user = create :au_user`

File: spec/controllers/vimeo_controller_spec.rb [hunk 3]
```diff
@@ -31,11 +47,12 @@
     end
 
     context "individual_feed" do
+      let(:feed) { FactoryGirl.create(:user_vimeo)}
```
Comment (vikshab) [inline spec/controllers/vimeo_controller_spec.rb]: same here `feed = create :feed`

File: spec/controllers/sessions_controller_spec.rb (hunk 2 above)
Comment (sojeri) [inline spec/controllers/sessions_controller_spec.rb:43]: Do you mean like `let(:user) { create :au_user }`, @vikshab?

File: spec/controllers/vimeo_controller_spec.rb (hunk 3 above)
Comment (sojeri) [inline spec/controllers/vimeo_controller_spec.rb]: Do you mean like `let(:feed) { create :user_vimeo }`, @vikshab?

**Comment by sojeri** [comment]
I'm so excited to see these VCR cassettes in action after we merge this PR! :D

File: spec/support/vcr_setup.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# spec/support/vcr_setup.rb
+VCR.configure do |c|
+  #the directory where your cassettes will be saved
+  c.cassette_library_dir = 'spec/vcr'
+  # your HTTP request service. You can also use fakeweb, webmock, and more
+  c.hook_into :typhoeus
+end
```
Comment (vikshab) [inline spec/support/vcr_setup.rb:7]: add a line)

File: spec/factories/feeds.rb [hunk 5]
```diff
@@ -5,4 +5,20 @@
...
+    avatar nil
+    platform "vimeo"
+    platform_feed_id 2
+  end
+
```
Comment (vikshab) [inline spec/factories/feeds.rb:23]: delete extra line

File: spec/controllers/sessions_controller_spec.rb (hunk 2 above)
Comment (vikshab) [inline spec/controllers/sessions_controller_spec.rb:43]: Yes, totes agree on that @drvonnjerryxlii

**Pull request merged**